guillotina/renderers.py

```diff
@@ -8,33 +8,35 @@
 from typing import Optional
 from zope.interface.interface import InterfaceClass
 
-import json
-import ujson
-
-
-class GuillotinaJSONEncoder(json.JSONEncoder):
-    def default(self, obj):
-        if isinstance(obj, complex):
-            return [obj.real, obj.imag]
-        elif isinstance(obj, datetime):
-            return obj.isoformat()
-        elif isinstance(obj, type):
-            return obj.__module__ + "." + obj.__name__
-        elif isinstance(obj, InterfaceClass):
-            return [x.__module__ + "." + x.__name__ for x in obj.__iro__]  # noqa
-        try:
-            iterable = iter(obj)
-        except TypeError:
-            pass
-        else:
-            return list(iterable)
-
-        if isinstance(obj, PermissionSetting):
-            return obj.get_name()
-        if callable(obj):
-            return obj.__module__ + "." + obj.__name__
-        # Let the base class default method raise the TypeError
-        return json.JSONEncoder.default(self, obj)
+import orjson
+
+
+def guillotina_json_default(obj):
+    if isinstance(obj, str):
+        if type(obj) != str:  # e.g, i18n.Message()
+            return str(obj)
+    if isinstance(obj, complex):
+        return [obj.real, obj.imag]
+    elif isinstance(obj, datetime):
```

Comment:
Is this still need with orjson?

Reply:
I'll remove it. I'll write a test to ensure orjson it serializes the date to the same format

Reply:
This isn't used when serializing resources because each field of the resource is serializes using the `value_serialize` (`IValueToJson` adapter). But it can be removed because orjson serializes datetime, date, and time instances to RFC 3339 format ("1970-01-01T00:00:00+00:00")